lib/platform/bitbucket-server/index.js

```diff
@@ -611,18 +611,38 @@ async function updatePr(prNo, title, description) {
 
 async function mergePr(prNo) {
   logger.debug(`mergePr(${prNo})`);
-  // TODO: Needs implementation
   // Used for "automerge" feature
-  const { version } = (await api.get(
```

Comment:
Is there not a need for `version` now?

Reply:
Ups, sorry. i missed that. I'll fix that.